src/Univ2LpOracle.sol

```diff
@@ -98,8 +98,8 @@ contract UNIV2LPOracle {
     modifier toll { require(bud[msg.sender] == 1, "UNIV2LPOracle/contract-not-whitelisted"); _; }
 
     // --- Data ---
-    uint8   public immutable dec0;  // Decimals of token0
-    uint8   public immutable dec1;  // Decimals of token1
+    uint256 public immutable normalizer0;  // Multiplicative factor that normalizes a token0 balance to a WAD; 10^(18 - dec)
```

Comment:
Do these need to be public? I can see the usefulness of returning orb decimals to people but I'm not sure these normalized values are useful externally.

Reply:
Was mostly following the previous convention; yeah, doesn't seem to be much point to exposing these.